crates/mipsy_web/src/worker.rs

```diff
@@ -78,6 +79,9 @@ pub enum WorkerRequest {
     // The struct that worker can obtain
     CompileCode(FileInformation),
     ResetRuntime(MipsState),
+    UpdateConfig(MipsyWebConfig),
+    // Toggle a breakpoiint at an address
```

Comment:
```suggestion
    // Toggle a breakpoint at an address
```
pwned

Reply:
honestly i saw this and couldnt be bothered so ty 😋